sycl/include/CL/sycl/aspects.hpp

```diff
@@ -49,6 +49,7 @@ enum class aspect {
   ext_oneapi_native_assert = 31,
   host_debuggable = 32,
   ext_intel_gpu_hw_threads_per_eu = 33,
+  bf16 = 34
```

Comment:
To reduce changes in future:
```suggestion
  bf16 = 34,
```

Reply:
Done